jacoco/src/main/java/de/kontext_e/jqassistant/plugin/jacoco/scanner/JacocoScannerPlugin.java

```diff
@@ -75,7 +75,8 @@ protected void configure() {
     @Override
     public boolean accepts(final FileResource item, String path, Scope scope) {
         try {
-            boolean accepted = path.endsWith(jacocoFileName) || (jacocoDirName.equalsIgnoreCase(item.getFile().toPath().getParent().toFile().getName()) && path.endsWith(".xml"));
+            boolean accepted = path.endsWith(jacocoFileName)
```

Comment:
Should belong to a different PR, its a different ticket.
I'll merge it anyway b/c in another branch I replaced the FileResource operations completely by string operations.